src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
ideal to refer to all instance parameters with the this keyword, e.g. this.isDone